vtysh/bgp_vty.c

```diff
@@ -4875,6 +4934,8 @@ cli_neighbor_timers_execute(char *vrf_name, int argc, const char *argv[])
     const struct ovsrec_bgp_neighbor *ovs_bgp_neighbor;
```

Comment:
If the neighbor is already included in the group, when setting values of timers for it, 
these  values synchronized with the values established for the group is done in function "cli_neighbor_timers_execute" by calling function "bgp_neighbor_param_timers".